docs/api.md

```diff
@@ -0,0 +1,72 @@
+# Whitehall API
+
+Whitehall is primarily a publishing application but it also provides an API used
+by some applications. This API is public and is served under
+[https://www.gov.uk/api/](https://www.gov.uk/api/governments).
+
+Whitehall is not the only application to provide APIs under this path. For
+example, [/api/content](https://www.gov.uk/api/content/government) is served by
+Content Store and [/api/search.json](https://www.gov.uk/api/search.json) is
+served by Rummager.
+[This Nginx file](https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/templates/publicapi_nginx_extra_config.erb)
+configures this behaviour.
+
+## Endpoints
+
+[`/api/governments`](https://www.gov.uk/api/governments) (no client available)
+
+Lists governments from newest to oldest. Includes title, start and end dates.
+
+[`/api/governments/:id`](https://www.gov.uk/api/governments/2015-conservative-government) (no client available)
+
+Shows the details for a single government.
+
+[`/api/organisations`](https://www.gov.uk/api/organisations) ([client](https://github.com/alphagov/gds-api-adapters/blob/0f24f4bc94ed1f8713c894b854c10ea867e6cf25/lib/gds_api/organisations.rb#L4-L6))
+
+Lists organisations. Includes title, status and relationships to other orgs.
+
+[`/api/organisations/:slug`](https://www.gov.uk/api/organisations/attorney-generals-office) ([client](https://github.com/alphagov/gds-api-adapters/blob/0f24f4bc94ed1f8713c894b854c10ea867e6cf25/lib/gds_api/organisations.rb#L8-L10))
+
+Shows the details for a single organisation.
+
+[`/api/world-locations`](https://www.gov.uk/api/world-locations) ([client](https://github.com/alphagov/gds-api-adapters/blob/0f24f4bc94ed1f8713c894b854c10ea867e6cf25/lib/gds_api/worldwide.rb#L4-L6))
+
+Lists world locations. Includes title, web URL and country code.
+
+[`/api/world-locations/:slug`](https://www.gov.uk/api/world-locations/afghanistan) ([client](https://github.com/alphagov/gds-api-adapters/blob/0f24f4bc94ed1f8713c894b854c10ea867e6cf25/lib/gds_api/worldwide.rb#L8-L10))
+
+Shows the details for a single world location.
+
+[`/api/world-locations/:slug/organisations`](https://www.gov.uk/api/world-locations/afghanistan/organisations) ([client](https://github.com/alphagov/gds-api-adapters/blob/0f24f4bc94ed1f8713c894b854c10ea867e6cf25/lib/gds_api/worldwide.rb#L12-L14))
+
+Lists worldwide organisations relating to a world location. Includes embassies,
+departments, sponsor organisations and details for offices such as addresses,
+telephone numbers and email addresses.
+
+[`/api/worldwide-organisations/:slug`](https://www.gov.uk/api/worldwide-organisations/department-for-international-trade-afghanistan) (no client available)
+
+Shows the details for a single worldwide organisation.
+
+## Consumers
+
+Please add your application to this list if you're using the API.
```

Comment:
Potentially could be obvious, but there are no instructions here about how to add your application to the list.